pubspec.lock

```diff
@@ -57,6 +57,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.2.0"
+  checked_yaml:
```

Comment:
Indeed, but I think I would consider ignite-cli an application right? Even if it is installed globally via pub, a CLI is an application in the end.

What do you think?

Reply:
I don't think so because the CLI is something that is installed as a dependency by other developers so the CLI should support the latest non-breaking versions of its dependencies.

[Stagehand](https://github.com/dart-archive/stagehand) for example did not track the `pubspec.lock` and neither does [pub](https://github.com/dart-lang/pub).

Reply:
Interesting, I have always considered CLIs as just another type of applications and that is why I thought it made sense checking the lock in the control.

But I think you are correct, this can be used as a dependency so it would probably not make sense to checking it. Maybe we can remove it on a follow up PR if @luanpotter agree